www/js/data.js

```diff
@@ -97,7 +97,7 @@ function getOrDefault(k, def) {
 }
 
 var USEROPTS = {
-    theme                : getOrDefault("theme", "/css/themes/slate.css"),
+    theme                : getOrDefault("theme", DEFAULT_THEME), // Set in head template
```

Comment:
Does this actually work?  I thought jade locals were local to the jade renderer.